index.html

```diff
@@ -1,1667 +1,2064 @@
-     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-     ga('create', 'UA-17146009-5', 'auto');
-     ga('send', 'pageview');
+<head>
+    <meta charset="utf-8">
+    <title>FOSSASIA | Asia's Open Technology Organization - Developing Open Source Software, Open Hardware, Open Design,
+        Open Data, Open Knowledge to Improve People's Life</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description"
+          content="FOSSASIA - Developing Open Source Software and Open Hardware to improve people's lives. Solutions for Event Management, Artificial Intelligence. FOSSASIA is Asia's Premier Open Technology Organization.">
+    <meta name="keywords" content="Open Source, Development, FOSSASIA, FOSS, Free and Open Source Software, Asia">
+    <meta name="author" content="FOSSASIA">
+    <meta property="og:title" content="FOSSASIA"/>
+    <meta property="og:type" content="website"/>
+    <meta property="og:image" content="https://fossasia.org/img.fossasia.png"/>
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@fossasia">
+    <meta name="twitter:title" content="FOSSASIA">
+    <meta name="twitter:description"
+          content="FOSSASIA Open Tech Organization - Developing Open Source, Open Hardware and Open Knowledge Tools with Asia's Premier Open Technology Community">
+    <meta name="twitter:creator" content="@fossasia">
+    <meta name="twitter:image" content="https://fossasia.org/img/fossasia.png">
+    <link rel=icon href="fossasia.ico">
+
+    <link href="css/bootstrap.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/flexslider.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/elegant-icons.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/font-awesome.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/pe-icon-7-stroke.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/lightbox.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/theme-lava.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/custom.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/custom-blog.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/bootstrap-theme.min.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href="css/text-overlay.css" rel="stylesheet" type="text/css" media="all"/>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300' rel='stylesheet' type='text/css'>
+    <script>
+        (function (i, s, o, g, r, a, m) {
```

Comment:
These are a lot of unwanted change. 
Please make changes relevant to your issue.